utils/runtime-fuzzer/src/generator/upload_program.rs

```diff
@@ -100,17 +102,45 @@ fn arbitrary_salt(u: &mut Unstructured) -> Result<Vec<u8>> {
 }
 
 fn config(
+    unstructured: &mut Unstructured,
     programs: Option<&NonEmpty<ProgramId>>,
     codes: Option<&NonEmpty<CodeId>>,
     log_info: Option<String>,
     current_balance: Balance,
-) -> StandardGearWasmConfigsBundle {
+) -> RandomizedGearWasmConfigBundle {
+    // Observation: with balance increased and no control instructions we get *a lot* of programs executed even if we're
+    // running fuzzer for just 5 minutes. Without balance increase disabling control results in gas limit exceeding quite often.
+    let no_control = unstructured.ratio(1, 4).unwrap();
```

Comment:
all `unstructured` usage logic for the `Randomized` config creation must be moved to the type's initializer function in `wasm-gen`.